lib/aot/compiler.cpp

```diff
@@ -2905,6 +3171,148 @@ class FunctionCompiler {
     stackPush(PHIRet);
   }
 
+  void compileAtomicCheckOffsetAlignment(llvm::Value *Offset,
+                                         llvm::IntegerType *IntType) {
```

Comment:
Where is the alignment value from `Instruction`?

Reply:
This checks wheather the the memory offset is align to TargetType, but not the real atomic instruction alignment.